docs/data/date-pickers/rfc-range-calendar.md

```diff
@@ -0,0 +1,70 @@
+---
+productId: x-date-pickers
+title: DX - RangeCalendar
+---
+
+# Range Calendar
+
+<p class="description">This page describes how people can use date range views with Material UI and how they can build custom date range views.</p>
+
+:::success
+Almost all the components of `RangeCalendar.*` are the same as their `Calendar.*` counterpart.
+:::
+
+## Usage with only days
+
+### Without Material UI
+
+The user can use the `<RangeCalendar.DaysGrid />`, `<RangeCalendar.DaysGridHeader />`, `<RangeCalendar.DaysGridHeaderCell />`, `<RangeCalendar.DaysGridBody />`, `<RangeCalendar.DaysWeekRow />` and `<RangeCalendar.DaysCell />` components to create a grid of days:
+
+```tsx
+import { RangeCalendar } from '@base-ui-components/react-x-date-pickers-pro/range-calendar';
+
+<RangeCalendar.Root value={value} onChange={setValue}>
+  {({ visibleMonth }) => (
+    <React.Fragment>
+      <div>
+        <RangeCalendar.GoToMonth target="previous">◀</RangeCalendar.GoToMonth>
+        {visibleMonth.format('MMMM YYYY')}
+        <RangeCalendar.GoToMonth target="next">▶</RangeCalendar.GoToMonth>
```

Comment:
Naming suggestion:
```suggestion
        <RangeCalendar.MonthNavigation direction="previous">◀</RangeCalendar.GoToMonth>
        {visibleMonth.format('MMMM YYYY')}
        <RangeCalendar.MonthNavigation direction="next">▶</RangeCalendar.GoToMonth>
```

Reply:
For the name of the component, on the POC I'm currently using `Calendar.SetVisibleMonth`.
The idea was that it's consistent with the state it updates (which is named `visibleDate`).
`Calendar.MonthNavigation` also works well.

For the `target` / `direction` prop, I think `direction` is problematic now that we can also pass a date directly to the component and not only "next" / "previous".

Reply:
I'm sort of on the same page about the naming here. Feels a bit off to name a component like you would name a function 🤔 I'm all in for keeping the `visibleMonth/Date/Year` in the name, but maybe `visibleMonthNavigation` (or so9mething in that area 😅 ) could work better?

Reply:
I think the most important is to be consistent with the other Base UI / Base UI X primitives.
I see that Base UI uses `<Menu.Trigger />` and I guess the big question is, if you have two triggers, would you name it `<MenuTriggerA />` (trigger is a verb) or `<Menu.ATrigger />` (trigger is a noun).

I'm totally fine with both.

Reply:
Would we have two triggers (or two very similar actions) for the exact same component?

I could imagine having something like `SomeComponent.Trigger` and `OtherComponent.Trigger` but I might very well be missing stuff

But in the scenario you are describing, I would lean towards `<Menu.ATrigger />`

Reply:
It was more a way to say that the current nomenclature `<Menu.Trigger />` on Base UI can be understood both as a noun and as a verb.
So when we try to create buttons with more detailed names, some of us might prefer to treat it as a noun or as a verb.